frontend/cypress/support/commands.ts

```diff
@@ -0,0 +1,41 @@
+declare namespace Cypress {
+  interface Chainable {
+    playEvents(num: number): Chainable<Response>;
+    checkEventPresence(): Chainable<Response>;
+    playUntilDate(date: string): Chainable<Response>;
+  }
+}
+
+Cypress.Commands.add('playEvents', (num) => {
+  let i = 0;
+  for (i; i<num; i++) {
+    cy.get('[data-cy=event]', {timeout: 2000}).then(() => {
+      cy.wait(500);
+      cy.get('[data-cy=event-continue]', {timeout: 2000}).first().click();
+    });
+    cy.wait(500);
+  }
+})
+
+Cypress.Commands.add('checkEventPresence', () => {
+  cy.get('cvd-events-layout').then(($events) => {
+    if ($events.children().length > 0) {
+      cy.playEvents(1);
+    }
+  });
+})
+
+Cypress.Commands.add('playUntilDate', (date) => {
+  // Requires medium speed and then plays for a month
+  var monthLoop = Array.from({length:35}, (v,k)=>k+1);
```

Comment:
Command is called "playUntilDate" but if you would use date further than 35 days it would not work. So maybe rename the functionto smthing like `playUntilNextMonth`